arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -281,7 +282,7 @@ config SW_VECTOR_RELAY
 config SW_VECTOR_RELAY_CLIENT
 	bool "Enable Software Vector Relay (client)"
 	default y if BOOTLOADER_MCUBOOT
```

Comment:
@ABOSTM this needs a modification. Rationale:

If you leave as is, then the feature will _always_ be `default y`, in a Cortex-M0 (without VTOR) platform, when we build an image to be loaded by MCUboot. We do not want that by default for this STM32 platform that has HW-based vector table remap mechanism. We want the feature to be available by not default.

So I propose:

config SW_VECTOR_RELAY_CLIENT
    bool "Enable Software Vector Relay (client)"
    depends on !CPU_CORTEX_M_HAS_VTOR
    default y if BOOTLOADER_MCUBOOT and !CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP
    help

Reply:
Thanks, Done.